requirements.txt

```diff
@@ -1 +1,2 @@
+future==0.16.0
 requests==2.3.0
```

Comment:
Yeah we probably should upgrade this. Many projects using `gapipy` are actually overwriting  `requests` with their own requirements. DRF is on `2.10`, tincan-dispatch on `2.9.1` etc, so us upgrading it shouldn't be a conflict